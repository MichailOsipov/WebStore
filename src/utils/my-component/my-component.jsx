import * as React from 'react';

export const MyComponent = () => (
    <div>My Component here!</div>
);
