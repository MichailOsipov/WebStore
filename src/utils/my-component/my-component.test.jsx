import * as React from 'react';
import {shallow, render} from 'enzyme';
import {MyComponent} from './my-component';

describe('MyComponent', () => {
    test('rendering', () => {
        const wrapper = shallow(
            <MyComponent />
        );
        expect(wrapper).toMatchSnapshot();
    });
    test('rendering', () => {
        const wrapper = render(
            <MyComponent />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
