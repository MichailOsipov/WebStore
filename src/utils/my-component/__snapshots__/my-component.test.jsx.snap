// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent rendering 1`] = `
<div>
  My Component here!
</div>
`;

exports[`MyComponent rendering 2`] = `
<div>
  My Component here!
</div>
`;
